# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1486238933 18000
#      Sat Feb 04 15:08:53 2017 -0500
# Node ID f81c759dfae7805ea0819f945fc6fd467b67a17e
# Parent  00ce7388d0449d5fbcc0306b0fd9beba49ac06e5
* bp-table.h: Also include <string>.

diff --git a/libinterp/parse-tree/bp-table.h b/libinterp/parse-tree/bp-table.h
--- a/libinterp/parse-tree/bp-table.h
+++ b/libinterp/parse-tree/bp-table.h
@@ -23,16 +23,17 @@ along with Octave; see the file COPYING.
 #if ! defined (octave_bp_table_h)
 #define octave_bp_table_h 1
 
 #include "octave-config.h"
 
 #include <list>
 #include <map>
 #include <set>
+#include <string>
 
 class octave_map;
 class octave_user_code;
 class octave_value_list;
 
 struct
 bp_type
 {
